leptonai/api/kv.py

```diff
@@ -0,0 +1,52 @@
+# Because most of the apis in kv do not return json, we will not use json_or_error here.
+from typing import Union
+
+from .connection import Connection
+
+
+def list_kvs(conn: Connection):
```

Comment:
I think previous convention is use singular (`list_deployment`, `list_secret` etc.)

Reply:
Got it, changed